addons/supplies/XEH_postInitServer.sqf

```diff
@@ -0,0 +1,19 @@
+#include "script_component.hpp"
+
+GVAR(playerTraits) = createHashMap;
+
+[QGVAR(storeTraits), {
+    GVAR(playerTraits) set _this;
```

Comment:
```suggestion
    params ["_uid", "_trait"];
    GVAR(playerTraits) set [_uid, _trait];
```
Readability, negligible performance difference (this isn't used often or in high-intensity scenarios).